### PR TITLE
[mailgun] fix 400 bad request happening with 'from' option

### LINKED ISSUE #12449: 400 Bad Request
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

```
When I use mailgun action, my Fastfile content is following:

 after_all do |lane|
    # This block is called, only if the executed lane was successful

      mailgun(
          postmaster: "sandboxxxx.mailgun.org",
          apikey: "key-xxx",
          to: "xx@qq.com",
          from: "xxx",
          reply_to: "xx@xx.com",
          success: true,
          message: "Mail Body",
          app_link: "http://www.myapplink.com",
          ci_build_link: "http://www.mycibuildlink.com",
        )
    end

  error do |lane, exception|
     puts "Failed Deployed!"
  end
  
  But something wrong with the error : 400 Bad Request.
  
 my resolutions: 
 1. i can send through Curl, but not in fastlane

  curl -s --user 'api:key-xx' https://api.mailgun.net/v3/sandboxxx.mailgun.org/messages -F from='Mailgun Sandbox <postmaster@sandboxxx.mailgun.org>' -F to='xiawei <xx@xx.com>' -F subject='Hello yinglong.zhang' -F text='Congratulations yinglong.zhang, you just sent an email with Mailgun!  You are truly awesome!'
{
  "id": "<20180504093236.1.12C288E9D4C1C4B5@sandboxxx.mailgun.org>",
  "message": "Queued. Thank you."
}%  

2. try below settings in mailgun

Security Settings For Outgoing Mail
Certificate Verification Not Required

it not works for me

```

### PR DESCRIPTION
**Pull request by joshdholtz**
Fixes #12449

## Wut
- Extra space

## Fix
- Deleted extra space

### CODE REVIEW

File: fastlane/lib/fastlane/actions/mailgun.rb [hunk 1]
```diff
@@ -111,7 +111,7 @@ def self.handle_params_transition(options)
       def self.mailgunit(options)
         sandbox_domain = options[:postmaster].split("@").last
         params = {
-          from: "#{options[:from]}<#{options[:postmaster]}>",
```
[the remark this answers is not in the document]
Comment (joshdholtz) [reply fastlane/lib/fastlane/actions/mailgun.rb]: 🤔

**Review by taquitos — APPROVED** [review]
Fix that space and then 🚢

**Comment by joshdholtz** [comment]
@taquitos I fixed it! Biggest PR ever 💪

**Comment by tanpengsccd** [comment]
get this error too in mailgun

**Pull request merged**